From: Cyrill Gorcunov <gorcunov@openvz.org>
Date: Fri, 10 Dec 2010 15:38:51 +0000 (+0300)
Subject: C/R: Restore variable name missed
X-Git-Url: http://git.openvz.org/?p=linux-2.6.32-openvz;a=commitdiff_plain;h=eb1f3c62b3fc652945dd108dde30cde8be7a6867

C/R: Restore variable name missed

In patch 66d247cf6dd51c6725cef2d01285398fe667ef6f
we've lost a part of variable name during the merge
which causes kernel to not build anymore. Fix it
by restoring the name we need.

Signed-off-by: Cyrill Gorcunov <gorcunov@openvz.org>
Signed-off-by: Pavel Emelyanov <xemul@openvz.org>
---

diff --git a/kernel/cpt/rst_files.c b/kernel/cpt/rst_files.c
index 75d591e..6f3963a 100644
--- a/kernel/cpt/rst_files.c
+++ b/kernel/cpt/rst_files.c
@@ -873,7 +873,7 @@ struct file *rst_file(loff_t pos, int fd, struct cpt_context *ctx)
 			err = rst_get_object(CPT_OBJ_FILE, pos, &fi, ctx);
 			if (err < 0)
 				goto err_out;
-			fixup_file_flags(file, rst_cred, &fi, 0, pos, ctx);
+			fixup_file_flags(file, rst_cred_origin, &fi, 0, pos, ctx);
 		}
 		get_file(file);
 		rst_creds_restore_current();
@@ -1036,7 +1036,7 @@ open_file:
 	}
 map_file:
 	if (!IS_ERR(file)) {
-		fixup_file_flags(file, rst_cred, &fi, was_dentry_open, pos, ctx);
+		fixup_file_flags(file, rst_cred_origin, &fi, was_dentry_open, pos, ctx);
 
 		if (S_ISFIFO(fi.cpt_i_mode) && !was_dentry_open) {
 			err = fixup_pipe_data(file, &fi, ctx);
